docs/src/pages/guides/styling.md

```diff
@@ -157,6 +157,14 @@ Now you're ready to write Tailwind! Our recommended approach is to create a `src
 @tailwind utilities;
 ```
 
+and then import the file to a layout you want to use Tailwind in:
```

Comment:
Remind me, does this work in an Astro component? I thought that in an Astro file you'll want to use a `<link>` tag to load the style, with something like: `<link href={Astro.resolve('../styles/global.css')} />`

Reply:
@FredKSchott true, as was mentioned by @drwpow above.